docs/getting-started/config/authentication.md

```diff
@@ -56,7 +56,15 @@ Open `packages/app/src/App.tsx` and below the last `import` line, add:
 import { githubAuthApiRef } from '@backstage/core-plugin-api';
 ```
 
-Search for `const app = createApp({` in this file, and below `apis,` add:
+Search for `const app = createApp({` in this file, and replace:
+
+```tsx title="packages/app/src/App.tsx"
+components: {
+    SignInPage: props => <SignInPage {...props} auto providers={['guest']} />,
+  },
```

Comment:
I think this needs to be aligned a bit, prettier should fix this for you.

Reply:
Seems prettier won't just fix this and I thought for sure the CI would also fail but it passed. Can we just make a quick adjustment any to this:

```tsx title="packages/app/src/App.tsx"
components: {
  SignInPage: props => <SignInPage {...props} auto providers={['guest']} />,
},
```

Reply:
fixed